src/worker.cc

```diff
@@ -842,6 +874,12 @@ void from_json(rapidjson::Document& doc, Options::Action action, Api& api) {
     }
   }
 
+  // TODO(nils): if we parse the costing options before the ignore_closures logic,
```

Comment:
then why not just move it back down where it was before?

Reply:
I actually wanted to have it right where the costing is created many lines up. 

I was just quickly testing if that's really the problem and left it that way to correlate it better to ignore_closures stuff since that's what makes it break and I don't immediately see why (and want to investigate another day). so it's more for future convenience that I left it here. I'll move it to the original place of course, if it's important to you